pom.xml

```diff
@@ -66,7 +66,7 @@
     <changelist>999999-SNAPSHOT</changelist>
     <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
     <jenkins.baseline>2.479</jenkins.baseline>
-    <jenkins.version>${jenkins.baseline}.1</jenkins.version>
+    <jenkins.version>${jenkins.baseline}.3</jenkins.version>
```

Comment:
I'm not sure this is related.
Plus, if we want to change the LTS, I'd go for a version still recommended.

Reply:
> I'm not sure this is related.

It is: https://github.com/jenkinsci/commons-lang3-api-plugin/blob/3a674c06072d8009fe769a048732116bc552eb3c/pom.xml#L19